modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view_{context}"]
+= Customizing a Developer catalog or its sub-catalogs using the form view
+
+You can customize a developer catalog by using the form view in the Web Console.
+
+.Prerequisites
+
+* An OpenShift web console session by a cluster administrator.
```

Comment:
Required: It sounds like there is a word missing here - maybe:

An OpenShift web console session created by a cluster administrator.
An OpenShift web console session with cluster administrator privileges.